src/foam/dao/Relationship.js

```diff
@@ -41,7 +41,7 @@ foam.CLASS({
       class: 'String',
       transient: true,
       hidden: true,
-      getter: function() {
+      factory: function() {
```

Comment:
I think the 'name' isn't actually used for anything, but it will break tooling because two relationships with the same source and target will have a name conflict. Instead, we could name it {sourceModel}{forwardName}{targetModel}{inverseName}, which remove the ambiguity. A factory is still better than a getter because it's more efficient and can be set. However, I think what we really want to change is junctionModel, which can already be set, as @noodlemoodle says. But I think we should have a new property called junctionName, which is just this part of junctionModlel: source + target + 'Junction', which can be overwritten, and then junctionModel will add the appropriate package to this.